wiki/Tournaments/YHC/3/en.md

```diff
@@ -0,0 +1,323 @@
+
+### Elimination stage 2
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **天使纷扰** ::{ flag=CN }:: | **6** | 3 | ::{ flag=CN }:: 笔不飞 不会打 | [#1](https://osu.ppy.sh/community/matches/109390391) |
+| 遇到HD睡大觉 ::{ flag=CN }:: | 3 | **6** | ::{ flag=CN }:: **孤儿收容所** | [#1](https://osu.ppy.sh/community/matches/109391200) |
+| **笔不飞 不会打** ::{ flag=CN }:: | **6** | 5 | ::{ flag=CN }:: null | [#1](https://osu.ppy.sh/community/matches/109278600) |
+| 6A级景点 ::{ flag=CN }:: | 2 | **6** | ::{ flag=CN }:: **孤儿收容所** | [#1](https://osu.ppy.sh/community/matches/109295823) |
+| **我倒杯水** ::{ flag=CN }:: | **6** | 5 | ::{ flag=CN }:: 遇到HD睡大觉 | [#1](https://osu.ppy.sh/community/matches/109279456) |
+| 天使纷扰 ::{ flag=CN }:: | -1 | **0** | ::{ flag=CN }:: **鲭西何时无缘进路** | *win by default* |
+
+### Elimination stage 1
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **遇到HD睡大觉** ::{ flag=CN}:: | **5** | 0 | ::{ flag=CN }:: null | [#1](https://osu.ppy.sh/community/matches/109161042) |
+| **我倒杯水** ::{ flag=CN }:: | **5** | 1 | ::{ flag=CN }:: 笔不飞 不会打 | [#1](https://osu.ppy.sh/community/matches/109142623) |
+| 孤儿收容所 ::{ flag=CN }:: | 4 | **5** | ::{ flag=CN }:: **鲭西何时无缘进路** | [#1](https://osu.ppy.sh/community/matches/109163592) |
+| 6A级景点 ::{ flag=CN }:: | 1 | **5** | ::{ flag=CN }:: **天使纷扰** | [#1](https://osu.ppy.sh/community/matches/109144755) |
+
+## Ruleset
+
+### General
+
+1. The Yuyuko's Hidden Cup Season 3 is a entertainment-oriented double elimination 4v4 team tournament, played on the osu! game mode.
+2. Beatmap scoring is based on ScoreV2.
+3. Based on the number of registration, **the size for a team is 7 players**.
+4. The beatmap list for each next round will be announced by the mappool selectors on the official live stream, on the weekend (Friday 12:00 - Monday 23:00) after the matches take place.
+5. The default match schedules for each round will be announced by the tournament managers in the information (main) sheet, on Sunday before the last matches take place. the captain could have a discussion with the opponent and confirm the match time, before Saturday 00:00 in the week.
+6. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.
+   - Custom skin elements must not be used to alter core gameplay elements or mechanics in unintended ways.
+7. If less than the minimum amount of required players are present at match time, the match can be postponed for up to 10 minutes. If after this period there are still not enough players for either team, a win by default will be declared for the side with the most members present.
+   - The minimum amount of required players is the amount of players needed to play a beatmap without any vacant spots in the lobby (i.e. 4 participants must be present for the match to begin). 
+8. Exchanging players between games is allowed without limitations.
```

Comment:
"exchanging" makes it sound like the teams exchange between themselves (at least to me)
```suggestion
8. Replacing players between games is allowed without limitations.
```